src/index.html

```diff
@@ -4,10 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet">
+
+    <link rel="shortcut icon" href="./images/icons/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+
+    <header class="header">
+
+      <div class="header__top">
+
+        <nav class="nav">
+          <div class="nav__icon"></div>
+          <div class="nav__menu">
+            <ul class="nav__items">
+              <li class="nav__item">Home</li>
+              <li class="nav__item">About Us</li>
+              <li class="nav__item">Gallery</li>
+              <li class="nav__item">Consultation</li>
+              <li class="nav__item">Testimonials</li>
+              <li class="nav__item">Contacts</li>
+              <li class="nav__item nav__item--phone">+1 234 555-55-55</li>
+              <li class="nav__item nav__item--text">call to order</li>
+              <li></li>
+            </ul>
+          </div>
+        </nav>
+        <div class="logo">
+          <img class="logo"
+          src="./images/icons/logo.png"
+          alt="Miami Condo Kings">
+        </div>
+        <div class="phone">
+          <div class="phone__number">+1 234 555-55-55</div>
+          <div class="phone__icon"></div>
+        </div>
+
+      </div>
+      <div class="header__text-container">
+        <h1 class="header__text">Miami Condo Kings</h1>
+      </div>
+    </header>
+
+    <div class="article">
+      <div class="article__left-col">
+        <h2 class="article__left-col-title">
+          Miami best Real estate <span>here</span>
+        </h2>
+      </div>
```

Comment:
```suggestion
      <h2 class="article__title">
        Miami best Real estate <span>here</span>
      </h2>
```
You can get rid of this strange-named `div` wrapper